onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -672,6 +672,7 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                OpSchema::Optional)
         .Input(9,
                "cache_indirection",
+               "This input is for CUDA EP only."
```

Comment:
It is probably not good to ask user to generate one model for CUDA, and another different model for ROCm. 

Is it possible to hipify the cuda kernel so that we can use same design?

Reply:
Not sure fastertransformer is hipifiable. CK does not consume. But I can just ignore this input at the moment because we don't need it to produce the correct result at all. So we don't need to produce different models due to this input.

So this is a just comment to note the fact that this input is only useful for some specific cuda kernels.

Reply:
Let's not add EP information in operator spec. You can add a comment instead.

Reply:
Changed to comment.